drop-in/src/main/java/com/adyen/checkout/dropin/DropInConfiguration.kt

```diff
@@ -47,6 +49,8 @@ import kotlinx.parcelize.Parcelize
 import java.util.Locale
 import kotlin.collections.set
 
+private const val INSTANT_PAYMENT = "INSTANT_PAYMENT"
```

Comment:
Can we move this to the companion?

Reply:
Yes and no. Both the `Builder` and the actual configuration use this field. If we put it in an `companion object` it has to be `public/internal` meaning other classes could access it. Now it's at least file private